src/rachanaGJun20/String/StringReplaceFun.java

```diff
@@ -0,0 +1,34 @@
+/*Count total special characters like _ , [ { ] } from every word, do not use any loop.
+Hint : Use replace method [Think about it, how it can help].
+
+Input :  Kr[i{s]hn}_a,Tech
+Output : total special characters are 6 */
+package rachanaGJun20.String;
+
+public class StringReplaceFun {
+
+	// method will print special char count in a given string
+	void countSpecialChar(String givenstr) {
+		
+
+		String newstr = givenstr;
+		newstr = newstr.replace("Kr", "");
```

Comment:
We should not write it like this. This is completely hardcoded... Your code wont work if name is maulik with same special character. and givenstr has no meaning then why to pass parameter if you want to say replace Kr with "" and so on...

Reply:
> Very good job. Code looks good to me.

Thanks :)

Reply:
> We should not write it like this. This is completely hardcoded... Your code wont work if name is maulik with same special character. and givenstr has no meaning then why to pass parameter if you want to say replace Kr with "" and so on...

yeah right got it.I have changed method calling , but not able to write other logic(without hardcoading) with out using loop. please suggest.